src/main/java/persistence/sql/dml/value/Value.java

```diff
@@ -0,0 +1,27 @@
+package persistence.sql.dml.value;
+
+import persistence.sql.ddl.utils.ReflectionUtil;
+
+import java.lang.reflect.Method;
+
+public class Value {
```

Comment:
조금 더 명확한 클래스 이름을 설정해주면 좋을 것 같아요 :)
```suggestion
public class ColumnValue {
```